src/Template/Pages/home.ctp

```diff
@@ -163,28 +163,52 @@ $cakeDescription = 'CakePHP: the rapid development PHP framework';
                     Our primary goal is to provide a structured framework that enables PHP users at all levels to rapidly develop robust web applications, without any loss to flexibility.
                 </p>
                 <ul>
-                    <li><a href="http://cakefoundation.org/">Cake Software Foundation</a>
-                    <ul><li>Promoting development related to CakePHP</li></ul></li>
-                    <li><a href="http://www.cakephp.org">CakePHP</a>
-                    <ul><li>The Rapid Development Framework</li></ul></li>
-                    <li><a href="http://book.cakephp.org/3.0/en/">CakePHP Documentation</a>
-                    <ul><li>Your Rapid Development Cookbook</li></ul></li>
+                       
+                     <h3>Help and Bug Reports</h3>
```

Comment:
This should really be split up into:

- Questions, Getting Help and the Community (more to the top) - also containing the book link possibly.
- Contribution and Bug Reports (more to the bottom)

... maybe helps a bit to get questions into the right channels.